common/build/eslint-config-fluid/printed-configs/strict.json

```diff
@@ -668,6 +668,32 @@
         "jsdoc/require-hyphen-before-param-description": [
             "error"
         ],
+        "jsdoc/require-jsdoc": [
+            "error",
+            {
+                "checkConstructors": true,
+                "checkGetters": true,
+                "checkSetters": true,
+                "contexts": [
+                    "TSEnumDeclaration",
+                    "TSInterfaceDeclaration",
+                    "TSTypeAliasDeclaration"
+                ],
+                "enableFixer": false,
+                "exemptEmptyConstructors": false,
+                "exemptEmptyFunctions": false,
+                "fixerMessage": "",
+                "publicOnly": true,
+                "require": {
+                    "ArrowFunctionExpression": false,
+                    "ClassDeclaration": true,
+                    "ClassExpression": false,
+                    "FunctionDeclaration": true,
+                    "FunctionExpression": false,
+                    "MethodDefinition": false
```

Comment:
Maybe we should also include the expressions?

Reply:
I'm not sure how often expressions would surface as the top-level item in a statement. Generally I would expect something of the form:

```ts
export const foo = function(...) { ... }
```

Where the documentation would be associated with the variable assignment, rather than the function. 

That said, it probably wouldn't hurt, and maybe I'm missing something. I'll do a local run with them turned on and see what I get.

Reply:
I stand corrected. eslint is smart enough to check the root item of the expression for corresponding docs, rather than the child node. 

In the example of:

```ts
const foo = function(bar: number, baz: string) { ... }
```

eslint is triggered for both `VariableDeclaration` and `FunctionExpression`.

Reply:
I also realized `VariableDeclaration` was missing from this list. I went ahead and enabled that too.